vectorstores/alloydb/options.go

```diff
@@ -62,10 +62,10 @@ func WithPassword(password string) Option {
 	}
 }
 
-// WithIPType sets the IpType field.
-func WithIPType(ipType string) Option {
+// WithPrivateIP sets the PrivateIP field.
+func WithPrivateIP(isPrivateIP bool) Option {
```

Comment:
Not sure a bool make sense here -- instead use WithPrivateIP and WithPublicIP to match the dialer opts.